pkg/kv/kvserver/raft_transport.go

```diff
@@ -566,11 +565,14 @@ func (t *RaftTransport) startProcessNewQueue(
 			return
 		}
 
-		if err := t.processQueue(toNodeID, ch, stream, class); err != nil {
+		if err := t.processQueue(ch, stream); err != nil {
 			log.Warningf(ctx, "while processing outgoing Raft queue to node %d: %s:", toNodeID, err)
 		}
 	}
-	err := t.stopper.RunAsyncTask(ctx, "storage.RaftTransport: sending messages", worker)
+	err := t.stopper.RunAsyncTask(ctx, "storage.RaftTransport: sending/receiving messages",
+		func(ctx context.Context) {
```

Comment:
Any opinions on the order of these wrappings? `RunAsyncTask`->`pprof.Do` or vice versa?

Reply:
This ordering looks good to me. We don't really care about the surrounding goroutine since it's short-lived, and this way it will be clear to folks that we're labelling the goroutine even if you don't know that profiler labels get propagated to children.